tools/vsextension/ProjectWizard/ImportEnclaveCommandPackage.vsct

```diff
@@ -0,0 +1,109 @@
+  <Extern href="vsshlids.h"/>
+
+  <!--The Commands section is where commands, menus, and menu groups are defined.
+      This section uses a Guid to identify the package that provides the command defined inside it. -->
+  <Commands package="guidImportEnclaveCommandPackage">
+    <!-- Inside this section we have different sub-sections: one for the menus, another
+    for the menu groups, one for the buttons (the actual commands), one for the combos
+    and the last one for the bitmaps used. Each element is identified by a command id that
+    is a unique pair of guid and numeric identifier; the guid part of the identifier is usually
+    called "command set" and is used to group different command inside a logically related
+    group; your package should define its own command set in order to avoid collisions
+    with command ids defined by other packages. -->
+
+    <Menus>
+      <Menu guid="guidImportEnclaveCommandPackageCmdSet" id="MySubMenu" priority="0x0100" type="Menu">
+        <Parent guid="guidImportEnclaveCommandPackageCmdSet" id="MyMenuGroup"/>
+        <Strings>
+          <ButtonText>Open Enclave Configuration</ButtonText>
+          <CommandName>Open Enclave Configuration</CommandName>
+        </Strings>
+      </Menu>
+    </Menus>
+
+    <!-- In this section you can define new menu groups. A menu group is a container for
+         other menus or buttons (commands); from a visual point of view you can see the
+         group as the part of a menu contained between two lines. The parent of a group
+         must be a menu. -->
+    <Groups>
+      <Group guid="guidImportEnclaveCommandPackageCmdSet" id="MyMenuGroup" priority="0x07FF">
+        <Parent guid="guidSHLMainMenu" id="IDM_VS_CTXT_PROJNODE"/>
+      </Group>
+      <Group guid="guidImportEnclaveCommandPackageCmdSet" id="MySubMenuGroup" priority="0x0100">
+        <Parent guid="guidImportEnclaveCommandPackageCmdSet" id="MySubMenu"/>
+      </Group>
+    </Groups>
+
+    <!--Buttons section. -->
+    <!--This section defines the elements the user can interact with, like a menu command or a button
+        or combo box in a toolbar. -->
+    <Buttons>
+      <!--To define a menu group you have to specify its ID, the parent menu and its display priority.
+          The command is visible and enabled by default. If you need to change the visibility, status, etc, you can use
+          the CommandFlag node.
+          You can add more than one CommandFlag node e.g.:
+              <CommandFlag>DefaultInvisible</CommandFlag>
+              <CommandFlag>DynamicVisibility</CommandFlag>
+          If you do not want an image next to your command, remove the Icon node /> -->
+      <Button guid="guidImportEnclaveCommandPackageCmdSet" id="ImportEnclaveCommandId" priority="0x0100" type="Button">
+        <Parent guid="guidImportEnclaveCommandPackageCmdSet" id="MySubMenuGroup" />
+        <Icon guid="guidImages" id="bmpPic1" />
```

Comment:
Is there a better icon? This one shows up as "1x".

Reply:
Removed